bundles/org.openhab.binding.squeezebox/src/main/java/org/openhab/binding/squeezebox/internal/handler/SqueezeBoxPlayerHandler.java

```diff
@@ -68,6 +68,7 @@
  * @author Patrik Gfeller - Timeout for TTS messages increased from 30 to 90s.
  * @author Mark Hilbush - Get favorites from server and play favorite
  * @author Mark Hilbush - Convert sound notification volume from channel to config parameter
+ * @author Mark Hilbush - Add like/unlike functionality
```

Comment:
Do you really need to add another author tag each time?

Reply:
No. I typically do it only for changes of some significance.